source/components/Header/components.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const Container = styled.header`
+  height: 4rem;
+  padding: 0 1.5rem;
+  position: relative;
+  z-index: 999;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #2c2c2c;
```

Comment:
Would be nice to add the color to the palette.

Reply:
Added this color to the palette as ghostgrey

Reply:
Cool stuff, liking it 😁